### PR TITLE
Remove button text updated

### LINKED ISSUE #61: Update to literal word `Remove` to replace with `x` icon
### Task

Currenty, to remove a project from the list, you have to click on the word `Remove`.

image

- [ ] Change the `Remove` to ❌ icon

### PR DESCRIPTION
**Pull request by sachinttn9663**
@haroon-sheikh FYI

Fixes #61

### CODE REVIEW

File: src/components/Filter/Filter.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ export default class Filter extends PureComponent {
             className="remove"
             onClick={this.props.onRemoveClick(project.name)}
           >
-            Remove
+            X
```
Comment (haroon-sheikh) [inline src/components/Filter/Filter.js:28]: I was actually thinking about making use of font-awesome icons instead.

Comment (haroon-sheikh) [reply src/components/Filter/Filter.js:28]: Also I think x icon may conflict with when the status is a failure. Let's change the icon to just a minus `-` icon.

**Review by haroon-sheikh — CHANGES_REQUESTED** [review]

File: src/components/Filter/Filter.js (hunk 1 above)
Comment (sachinttn9663) [reply src/components/Filter/Filter.js:28]: Ok, But I think react-feather icons would be good to use compared to font-awesome

Comment (haroon-sheikh) [reply src/components/Filter/Filter.js]: Yeah that's fine with me.

**Comment by haroon-sheikh** [comment]
Thanks @sachinttn9663 for contributing.

**Pull request merged**